spec/cucumber/cli/profile_loader_spec.rb

```diff
@@ -42,6 +42,16 @@ def loader
         given_cucumber_yml_defined_as yml
         expect(loader.args_from('default')).to eq ['--format','pretty','features/sync_imap_mailbox.feature:16:22']
       end
+
+      it 'correctly parses a profile that uses tag expressions' do
```

Comment:
I would also add an example using `'` around the argument containing blanks, for instance `"--format pretty features --tags 'not @jruby'"` (which also avoids the path separator problem), to be very sure that both `"` and `'` can be used to define arguments with blanks in them.